config/_electrum_urls/mainnet

```diff
@@ -1 +1,3 @@
-ssl://electrum.blockstream.info:50002
\ No newline at end of file
+wss://electrumx-server.tbtc.network:8443
+wss://electrum.boar.network:2083
+ssl://electrum.blockstream.info:50002
```

Comment:
P2P set up their Electrum server. It is available at:
wss://bitcoin.threshold.p2p.org:50004
ssl://bitcoin.threshold.p2p.org:50002

Reply:
It wasn't available the last time I checked. I'll test it again.